tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/RealMetricName.java

```diff
@@ -50,6 +50,11 @@ public String safeName() {
         return safeTags;
     }
 
+    @Override
+    public String toString() {
+        return "MetricName{safeName=" + safeName + ", safeTags=" +  safeTags + '}';
```

Comment:
should we condense this, just make it JSON (like) for simplicity?

Reply:
This matches ImmutableMetricName, I'd prefer to have RealMetricName match the other implementation for now, I'm not sure there's enough benefit to changing the structure to something json-like for it to be worth the change. Thoughts?

Reply:
as-is is fine by me